scripts/rspec/rspec.ps1

```diff
@@ -59,6 +59,7 @@ $categoriesMap = @{
     "BUG" = "Bug";
     "CODE_SMELL" = "Code Smell";
     "VULNERABILITY" = "Vulnerability";
+    "SECURITY_HOTSPOT" = "Security Hotspot";
```

Comment:
I am not sure you still need this line with the change you did on line 193

Reply:
it's used for the category in `RspecStrings.resx`. Which means that the category is not LTS compatible. @dbmeneses  - is this a problem?

Reply:
I'm not sure whats the link between the script and the resource file, but I don't see this map being used anywhere else except on line 195.